x/inflation/v1/keeper/integration_test.go

```diff
@@ -65,15 +65,15 @@ var _ = Describe("Inflation", Ordered, func() {
 					s.CommitAfter(time.Hour * 25) // End Epoch
 				})
 
-				It("should allocate funds to usage incentives", func() {
+				It("should not allocate funds to usage incentives (deprecated)", func() {
 					actual := s.app.BankKeeper.GetBalance(s.ctx, addr, denomMint)
 
 					provision := s.app.InflationKeeper.GetEpochMintProvision(s.ctx)
 					params := s.app.InflationKeeper.GetParams(s.ctx)
-					distribution := params.InflationDistribution.UsageIncentives
+					distribution := params.InflationDistribution.UsageIncentives //nolint:staticcheck
 					expected := (provision.Mul(distribution)).TruncateInt()
 
-					Expect(actual.IsZero()).ToNot(BeTrue())
+					Expect(actual.IsZero()).To(BeTrue())
```

Comment:
nit: I think using `BeZero()` would be more straight forward in these cases
```suggestion
					Expect(actual).To(BeZero())
```

Reply:
Sorry this actually broke the tests because of the `nil` value inside of the amount instead of 0 :sweat_smile: